docs/nitpick-exceptions

```diff
@@ -16,3 +16,5 @@ py:class a set-like object providing a view on D's keys
 py:class an object providing a view on D's values
 py:class None.  Remove all items from D.
 py:class a shallow copy of D
+py:class v, remove specified key and return the corresponding value.
+py:class None.  Update D from dict/iterable E and F.
```

Comment:
maybe?
```suggestion
py:class None.  Update D from dict/iterable E and F.
py:obj reference target not found: iterator
```

Reply:
iterator is one I introduced. We should patch the doc string to not use it.